dev/com.ibm.ws.microprofile.reactive.messaging_fat/fat/src/com/ibm/ws/microprofile/reactive/messaging/fat/suite/PlaintextTests.java

```diff
@@ -35,6 +36,7 @@
                 KafkaCustomSerializerTest.class,
                 KafkaPartitionTest.class,
                 KafkaSharedLibTest.class,
+                ReactiveStreamsTckTest.class,
```

Comment:
How many tests are there in the TCK? I think i would prefer the TCK to be a separate bucket??

Reply:
Not many, it's a couple of classes though each class has quite a few methods.

I think it makes more sense in here. It's not a large TCK like the ones that verify each of the MP components and it runs pretty quickly.